src/data/actions/assets.js

```diff
@@ -34,14 +47,9 @@ export const deleteAssetSuccess = (assetId, response) => ({
   response,
 });
 
-export const assetXHRFailure = response => ({
-  type: assetActions.ASSET_XHR_FAILURE,
-  response,
-});
-
-export const deleteAsset = (assetsParameters, assetId) =>
+export const deleteAsset = (assetId, courseDetails) =>
```

Comment:
Removed assetParameters as it was no longer being used by this function.  Updated usages across files to handle the function change.

CC @efischer19